src/main/java/edu/ucla/library/bucketeer/handlers/AbstractBucketeerHandler.java

```diff
@@ -50,6 +50,8 @@ protected void sendMessage(final Vertx aVertx, final JsonObject aJsonObject, fin
                         final String messageCode = CodeUtils.getCode(replyException.failureCode());
                         final String details = replyException.getMessage();
 
+                        // FIXME: this throws IndexOutOfBoundsException if the reply failed with a code whose message
```

Comment:
Did you mean to leave this FIXME in? Seems it'd be good to test for this now (and change the way the error is logged)?

Reply:
I did mean to leave it in, but I can also fix it. Will do